task/s2i-java-binary-build/0.1/README.md

```diff
@@ -0,0 +1,51 @@
+# S2i Java Binary Build
+
+Sometimes, we won't be able to use the source to image approach,
+because we are previously generating the binary using a build automation service.
+In this situation, we can still use the builder images for runtime.
+
+[S2i Java Binary Build](./s2i-java-binary-build.yaml) task takes `binary` file as an input and create an
+`image` of that application and pass the image URL to the `result` of the task.
+
+**NOTE**: This task requires an openshift cluster to run.
```

Comment:
I think this is kind of prerequisite because this task won't run without openshift cluster :thinking: so can we add it in pre-requisites section something like that ?

Reply:
Yes, makes sense. Thanks :slightly_smiling_face: